corehq/apps/es/case_search.py

```diff
@@ -420,9 +420,10 @@ def wrap_case_search_hit(hit, include_score=False):
     `corehq.apps.es.case_search.ElasticCaseSearch._from_dict`.
 
     The "case_properties" list of key/value pairs is converted to a dict
-    and assigned to `case_json`. 'Secial' case properties are excluded
+    and assigned to `case_json`. 'Special' case properties are excluded
     from `case_json`, even if they were present in the original case's
-    dynamic properties.
+    dynamic properties, except of the COMMCARE_CASE_COPY_PROPERTY_NAME
+    property.
```

Comment:
`COMMCARE_CASE_COPY_PROPERTY_NAME` should be a reserved property on a case, but it should still show up when wrapping the case result from CaseSearchES since it's needed to determine whether the particular case is a copy.

Reply:
Does it belong in the `SPECIAL_CASE_PROPERTIES` list, then?  Looking at that definition I see
https://github.com/dimagi/commcare-hq/blob/77c2ca5dcf6f5aa02e51da2523b44f5bec59aa31/corehq/apps/case_search/const.py#L38-L41
So it looks like it's for things to be copied from the case object into the set of case properties when sending cases to the case search index.  Since this property is a normal case property that's already there, I'm not sure it's a good fit.

What do you mean by a reserved property?  One that can't be set by users (we can't control what they do in forms, unfortunately)?  One that they can't search for in case search and the CLE?  One that's hidden in reports?